Util/Sf2CompatUtil.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+/*
+ * This file is part of the Symfony CMF package.
+ *
+ * (c) 2011-2015 Symfony CMF
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Cmf\Bundle\CoreBundle\Util;
+
+final class Sf2CompatUtil
```

Comment:
this utility is quite similair to the one provided by the SonataCoreBundle. However, I don't like depending on another package for something like this. Including such a utility in each bundle means we can make the class as small as possible, by only including the needs for one bundle.